src/big/big_decimal.cr

```diff
@@ -282,6 +282,6 @@ end
 struct Float
   # Casting from `Float` is not supported due to precision loss risks. This call fails at compile time.
```

Comment:
>[...] This call fails at compile time.

no longer true.

Reply:
Fixed this comment and also a similar  one earlier in the file.